app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def logged_in?
```

Comment:
this method was already in application controller, please remove one of those two.